static/pages/config.html

```diff
@@ -29,9 +34,10 @@ <h3>Firmware Version : </h3>
 <div id="userAgent"></div>
 <hr>
 
+<p id="last-error" class="error hidden"></p>
```

Comment:
Please use camelcase for ids and kebab-case for classes
```suggestion
<p id="lastError" class="error hidden"></p>
```